content/en/blog/2022/prow-and-tide-for-kubernetes-contributors/prow-and-tide-for-kubernetes-contributors.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: Prow and Tide for Kubernetes Contributors
+date: 2022-05-31
+slug: prow-and-tide-for-kubernetes-contributors
+---
+
+**Authors:** [Chris Short](https://github.com/chris-short)
+
+---
+
+In my work in the Kubernetes world, I look up a label or Prow command often. The systems behind the scenes ([Prow](https://prow.kubernetes.io/) and [Tide](https://pkg.go.dev/k8s.io/test-infra/prow/cmd/tide#section-readme)) are here to help Kubernetes Contributors get stuff done.
+
+Labeling which SIG, WG, subproject, etc. is as important as the issue or PR having someone assigned. To quote [the docs](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md), "Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md) component for managing a pool of GitHub PRs that match a given set of criteria. It will automatically retest PRs that meet the criteria ('tide comes in') and automatically merge them when they have up-to-date passing test results ('tide goes out')."
+
+Something that really helps move things along is the [label_sync](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#intro) tool along with the logic Prow and Tide bring to the community.
```

Comment:
Why?   How does label sync help?  This sentence feels like it belongs further down in the article, where you're talking about managing labels.

Reply:
agree with expanding or moving it down below, label_sync ensures that our labels are applied uniformly across orgs and repos.